<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calendar_create</class>
 <widget class="QDialog" name="calendar_create">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>175</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>* Summary:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="summary_lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="description_lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="location_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_pushButton">
      <property name="text">
       <string>Create Calendar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
